integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -1005,19 +1035,26 @@ void testUpdateWDTVersionUsingMultipleAuxiliaryImages() {
         domainUid, miiAuxiliaryImage4, miiAuxiliaryImage5, wdtDomainNamespace);
     createDomainAndVerify(domainUid, domainCR, wdtDomainNamespace,
         adminServerPodName, managedServerPrefix, replicaCount);
+    
+    //create router for admin service on OKD in wdtDomainNamespace
+    adminSvcExtHost = createRouteForOKD(getExternalServicePodName(adminServerPodName), wdtDomainNamespace);
```

Comment:
Can it be moved to beforeAll() method so that  every method has access to this variable.

Reply:
No, it can not because in this class we use varied auxiliary images to create domain. adminSvcExtHost is per admin service/domain.